eng/Version.Details.xml

```diff
@@ -1,36 +1,36 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="6.0.200-rtm.22078.8" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="6.0.200-rtm.22107.3" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>e950d071463b02978f13d5324aeadd0eb9a51cfc</Sha>
+      <Sha>c506cfba13ac8861afdaaeadacb34fe7f54cde88</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.2-mauipre.1.22075.3" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.3" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
```

Comment:
@Redth  @jonathanpeppers  I see thisis the issue because 6.0.3 is not signed but our base dotnet sdk is. We can either remove the signature of the dotnet on CI or we can use a different, unsigned sdk version.

Reply:
Removing the signing in the yaml for now.

Reply:
The builds used here will almost never be signed.

`6.0.2-mauipre.1.22075.3` was only signed because it was a build inserted into VS, and that only happens a couple times during a release.